generated/nifake/unit_tests/test_session.py

```diff
@@ -680,6 +680,16 @@ def test_device_method_not_exist_on_repeated_capability_error(self):
             except AttributeError:
                 pass
 
+    def test_repeated_capabilities_list(self):
```

Comment:
Missing: test with invalid values passed for repeated capabilities.

Reply:
We actually test all the possibilities more comprehensively in the _converters unit tests, so remove all but one here. Just test that the repeated capabilities are passed along correctly.